packages/daemon/src/daemon.js

```diff
@@ -746,23 +687,25 @@ const makeDaemonCore = async (
    * @param {string} id
    * @param {import('./types.js').Context} context
    */
-  const makeControllerForId = async (id, context) => {
+  const evaluateFormulaForId = async (id, context) => {
```

Comment:
Consider: `makeFormulaForId()` or something along those lines. I'm not sold on the `FormulaForId` suffix, which doesn't seem to illuminate what distinguishes this function from `evaluateFormula` / `makeFormula`, since both take the id as an argument.

Reply:
Good points all around.

The first name I considered was `makeFormulaForId` (just removing the `Controller`) and I hesitated because it makes a value and does not make a formula. It uses the formula to make a value, which struck me as indistinguishable from evaluating a node in any other abstract syntax, which led me to `evaluateFormula`.

The distinction between `evaluateFormula` vs `evaluateFormulaForId` is going to haunt me. The latter presumes that it can find the formula for the identifier, and the former requires that to be spelled out, and all roads flow to `evaluateFormula`.

I’m going to merge for now and continue to think and discuss this with you.